test_integration_app/main.php

```diff
@@ -0,0 +1,107 @@
+<?php
+    require_once dirname(__DIR__) . '/vendor/autoload.php';
+    require_once dirname(__DIR__) . '/test_integration_app/events_api/test_events_api.php';
+    require_once dirname(__DIR__) . '/test_integration_app/decisions_api/test_decisions_api.php';
+    require_once dirname(__DIR__) . '/test_integration_app/workflows_api/test_workflows_api.php';
+    require_once dirname(__DIR__) . '/test_integration_app/score_api/test_score_api.php';
+    require_once dirname(__DIR__) . '/test_integration_app/verifications_api/test_verification_api.php';
+    require_once dirname(__DIR__) . '/test_integration_app/psp_merchant_api/test_psp_merchant_api.php';
+
+    // set api_key and account_id as Environment variables
+    putenv("api_key=PUT_A_VALID_API_KEY");
```

Comment:
looks like this command with SET an env variable, but we need to read it from the environemnt. So, what is the idea here?

Reply:
Fixed this issue and removed this line.
We can set the env variable through Windows Command using the following command:
$env:account_id="PUT_A_VALID_ACCOUNT_ID"
$env:api_key="PUT_A_VALID_API_KEY"